api/external/infra_proxy/infra_proxy.proto

```diff
@@ -250,11 +250,6 @@ service InfraProxy {
     option (chef.automate.api.iam.policy).resource = "infra:infraServers:{server_id}:orgs:{org_id}:nodes";
     option (chef.automate.api.iam.policy).action = "infra:infraServers:get";
   };
-  rpc GetAffectedNodes (infra_proxy.request.AffectedNodes) returns (infra_proxy.response.AffectedNodes) {
```

Comment:
Explain more on what affectedNodes means in this context.  May be adding a couple of lines to the Description will do.

Reply:
In an earlier discussion, every object like cookbooks, roles, and environments has tabs to show how many nodes are using these objects(affected by these objects).  As of now, we have implemented the search API for fetching the objects. we can directly get these objects mentioned in the PR description.

No caption needed as we are removing this API.